customer-testing/src/main/java/no/unit/nva/customer/testing/CustomerDataGenerator.java

```diff
@@ -96,6 +99,13 @@ public static UUID randomIdentifier() {
         return UUID.randomUUID();
     }
 
+    public static URI randomCristinOrgId() {
```

Comment:
this method is written in two places, once for the customer service and once for the user service. These services do not share any logic.